### PR TITLE
Todos

### PR DESCRIPTION
**Pull request by Chiragmodi01**
### Add , edit and delete todos

ignore netlify failed checks

### CODE REVIEW

**Review by harshitpaliwal95 — APPROVED** [review]
file structure is good 
code is clean you can merge good work

**Comment by aurobindoGupta** [comment]
- some checks are failing plz check
- code looks good.

File: src/screens/UserOnboarding/UserOnboarding.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.UserOnboarding {
+    height: 100vh;
+    width: 100vw;
+    color: var(--white);
+    padding: 1rem;
+    box-sizing: border-box;
+    background-color: rgba(0,0,0,.28);
```
Comment (aurobindoGupta) [inline src/screens/UserOnboarding/UserOnboarding.css:7]: variable

**Review by aurobindoGupta — APPROVED** [review]

**Review by agrawalshanu708 — COMMENTED** [review]
file structure is good 
UI looks clean

**Pull request merged**